workdir/openapi.yaml

```diff
@@ -10526,6 +10545,13 @@ paths:
         schema:
           example: 30d
           type: string
+      - description: Indicates if the device is covered under active support contract
```

Comment:
- description: Indicates if the device is covered under active support contract
or not

Alternative:

- description: Indicates if the device is covered under active support contract